metadata/indices/h3_hexes_level_5.md

```diff
@@ -0,0 +1,69 @@
+# Title
+
+Utah H3 Hexes Level 5
+
+## ID
+
+4e8fb736-5e1a-4c84-8683-88b0fcc15768
+
+## Brief Summary
+
+## Summary
+
+The [H3 indexing system](https://h3geo.org/docs) provides a standardized and high-performance grid for aggregating data at various levels. This layer contains the spatial representation of the hex IDs that cover the state of Utah for resolution level 5.
+
+## Description
+
+### What is the dataset?
+
+The H3 Spatial Index (created by Uber) helps users aggregate point data using a reusable common grid with consistent geometry. H3 uses highly optimized math routines to assign lat/long coordinates an ID corresponding to a polygon (almost always hexes) at a given [resolution level](https://h3geo.org/docs/core-library/restable). The resulting hexes can be used to aggregate data and display spatial statistics.
+
+### What is the purpose of the dataset?
+
+This dataset is suitable for normal GIS point-in-polygon operations, however, UGRC recommends using the [H3 API](https://h3geo.org/docs/api/indexing) to assign a hex ID to your points, aggregating/analyzing your points based on ID, and then joining your aggregated data to this layer on the hex IDs.
```

Comment:
```suggestion
This dataset is suitable for normal GIS point-in-polygon spatial operations. However, UGRC recommends that you use the [H3 API](https://h3geo.org/docs/api/indexing) to assign a hex ID to your points, then aggregate and analyze your points based on those IDs, and finally join your aggregated data table to this layer on the hex IDs to get a spatial representation. This can be much faster for large datasets than spatial point-in-polygon operations.
```